### PR TITLE
Add firmware version to TPMInfo for TPM 2.0 devices.

### PR DESCRIPTION
**Pull request by twitchy-jsonp**
Not 100% on the API for this, PTAL.

### CODE REVIEW

File: attest/attest.go [hunk 1]
```diff
@@ -224,11 +224,16 @@ var (
 
 // TPMInfo contains information about the version & interface
 // of an open TPM.
+// FWVersionMajor and FWVersionMinor describe the firmware version of
```
Comment (ericchiang) [inline attest/attest.go]: Comment should probably be over the fields not the struct?

Are there any links or background you can add to this comment? E.g. what are reasonable values to take? Could these potentially indicate vulnerable TPMs?

File: attest/attest.go [hunk 2]
```diff
@@ -224,11 +224,16 @@ var (
...
+// the TPM, but are only available for TPM 2.0 devices.
 type TPMInfo struct {
 	Version      TPMVersion
 	Interface    TPMInterface
 	VendorInfo   string
 	Manufacturer TCGVendorID
+
+	FWVersionMajor int
```
Comment (ericchiang) [inline attest/attest.go]: nit: Maybe don't abbreviate? These are going to be used sufficiency rare and I don't get FW -> Firmware

```
FirmwareVersionMajor int
FirmwareVersionMinor int
```

File: attest/attest.go (hunk 1 above)
Comment (twitchy-jsonp) [reply attest/attest.go]: Done. Not really, the values are vendor specific.

File: attest/attest.go (hunk 2 above)
Comment (twitchy-jsonp) [reply attest/attest.go]: Done.

**Pull request merged**